pom.xml

```diff
@@ -38,7 +38,7 @@
     <errai.marshalling.very_short_names>false</errai.marshalling.very_short_names>
     <!-- The version greater than 1.0.0.GA is not compatible with GWT 2.8.x -->
     <version.javax.validation>1.0.0.GA</version.javax.validation>
-    <wildfly.bootable.jar.gav>org.wildfly:wildfly-galleon-pack:${version.org.wildfly}</wildfly.bootable.jar.gav>
+    <wildfly.bootable.jar.gav>org.jboss.eap:wildfly-galleon-pack:4.0.0.GA-redhat-00002</wildfly.bootable.jar.gav>
```

Comment:
this should be defined/specified in productized profile i think

Reply:
True. I will look into it.

Reply:
I have a comment regarding the prod XP version that you are using. Prod version 4.0.0.GA-redhat-00002 is EAP XP4 which is not released yet (not available in MRRC), EAP QE is still testing it.

The latest prod version of XP public available is 3.0.0.GA-redhat-00012, which is on top of EAP 7.4.3, so I believe this is the version we should use currently.

Reply:
I have a comment regarding the prod XP version that you are using. Prod version 4.0.0.GA-redhat-00002 is EAP XP4 which is not released yet (not available in MRRC), EAP QE is still testing it.

The latest prod version of XP public available is 3.0.0.GA-redhat-00012, which is on top of EAP 7.4.3, so I believe this is the version we should use currently.

Reply:
@rgdoliveira Right, I have access for the XP4 and the goal is to update to XP4. This means this PR might be blocked until it is released, but would be helpful to run tests against the 4.0.0.GA-redhat-00002 if that is possible?

Reply:
@Rikkola I agree, I guess this PR needs to be on hold until XP4 is released to MRRC.
Regarding running the tests against prod build 4.0.0.GA-redhat-00002, do you mean CI? (cc @Ginxo)

Reply:
@rgdoliveira Yes manual testing passed, but it is more or less limited. So would be good to pick up any issues before the final is released and we might get it fixed for us.

Reply:
@Rikkola do we keep this PR as draft? Thanks